packages/cli/src/commands/site/deploy/siteDeployCommand.mts

```diff
@@ -59,7 +59,7 @@ export default async function siteDeployCommand(
   if (version != null) {
     siteVersion = version;
   } else {
-    siteVersion = await findAutoVersion(gitTagPrefix);
+    siteVersion = findAutoVersion(gitTagPrefix);
```

Comment:
`autoVersion` was labeled async but didn't do anything async so removed that and here for accuracy.